db/seeds.rb

```diff
@@ -1,7 +1,27 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+WHERE     = ['Pension Provider', 'Citizens Advice', 'My Employer', 'TV Advert'].freeze
+PENSIONS  = ['Hargreaves Lansdown', 'Aeon', 'Standard General', 'TSB'].freeze
+LOCATIONS = %w(Dublin Belfast Reading Essex Slough Somerset Cork).freeze
+PARTNER   = %w(CAS TPAS NICAB CITA).freeze
+
+puts 'Deleting existing `WhereDidYouHear` records'
+WhereDidYouHear.delete_all
+
+puts 'Creating new `WhereDidYouHear` records'
+
+100.times do
+  attributes = {
+    given_at: rand(-10.days..10.days).seconds.ago,
+    location: LOCATIONS.sample,
+    delivery_partner: PARTNER.sample,
+    where: WHERE.sample,
+    pension_provider: PENSIONS.sample
+  }
+
+  attributes.except!(:pension_provider) unless attributes[:where] == 'Pension Provider'
+
+  WhereDidYouHear.create!(attributes)
+  print '.'
+end
+
+puts
+puts 'done!'
```

Comment:
How so? It creates 100 records of varying data from a closed set for the purposes of testing / demonstrating this feature during development / staging. Seems just like seeds to me.